antarest/study/storage/variantstudy/model/command/create_binding_constraint.py

```diff
@@ -75,13 +75,13 @@ class BindingConstraintProperties(BaseModel, extra=Extra.forbid, allow_populatio
     enabled: bool = True
     time_step: BindingConstraintFrequency = BindingConstraintFrequency.HOURLY
     operator: BindingConstraintOperator = BindingConstraintOperator.EQUAL
-    comments: t.Optional[str] = None
-    filter_year_by_year: t.Optional[str] = None
-    filter_synthesis: t.Optional[str] = None
+    comments: t.Optional[str] = ""
+    filter_year_by_year: t.Optional[str] = ""
+    filter_synthesis: t.Optional[str] = ""
 
 
 class BindingConstraintProperties870(BindingConstraintProperties):
-    group: t.Optional[str] = None
+    group: t.Optional[str] = ""
```

Comment:
I don't really like these changes on default values. But I'll let Laurent choose.

Reply:
Other than that I like the fact that we kept back-end validation on the "group" field.

Reply:
indeed it's better for the direct users of the API to be informed about the error yes 👍🏼

Reply:
I prefer to use "default" as a default value yes.

Reply:
I agree it makes more sense, i will update it